contrib/internal/httptrace/options.go

```diff
@@ -0,0 +1,23 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2022 Datadog, Inc.
+
+package httptrace
+
+import "os"
+
+type config struct {
+	ipHeader string
+}
+
+var (
+	clientIPHeaderEnvVar = "DD_CLIENT_IP_HEADER"
```

Comment:
```suggestion
	clientIPHeaderEnvVar = "DD_TRACE_HTTP_CLIENT_IP_HEADER"
```

Reply:
Yes, please. Thanks for doing that. We've had a lot of issues with inconsistencies in naming env. vars previously, especially as things span cross-language, so let's try and change it. `DD_` stuff is many times agent related and client stuff is always `DD_TRACE_`

Reply:
Suggestion has been approved, renaming to `DD_TRACE_CLIENT_IP_HEADER`